pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -40,6 +41,40 @@ var (
 
 	  # Add the 'edit' role to serviceaccount1 for the current project
 	  %[1]s edit -z serviceaccount1`)
+
+	addRoleToUserLongDesc = templates.LongDesc(`
+	  Add a role to users or service accounts for the current project
+
+	  This command allows you to grant a user access to specific resources and actions within the current project, by assigning them to a role. It creates or modifies a RoleBinding referencing the specified role adding the user(s) or serviceaccount(s) to the list of subjects. The command does not require that the matching role or user/serviceaccount resources exist and will create the binding successfully even when the role or user/serviceaccount do not exist or when the user does not have access to view them.
+
+	  If the --rolebinding-name argument is supplied, it will look for an existing rolebinding with that name. The role on the matching rolebinding MUST match the role name supplied to the command. If no rolebinding name is given, a default name will be used. When --role-namespace argument is specified as a non-empty value, it MUST match the namespace for the current project. When role-namespace is specified, the rolebinding will reference a namespaced Role. Otherwise, the rolebinding will reference a ClusterRole resource.
```

Comment:
`it MUST match the namespace for the current project` `->`
`it MUST match the current namespace`